src/test/kotlin/minesweeper/domain/CellManagerTest.kt

```diff
@@ -44,12 +46,46 @@ class CellManagerTest {
             Position(1, 2),
             Position(1, 3)
         )
-        val cellManager = CellManager(positions.map { it.toCell() }.toMutableSet())
+        val cellManager = CellManager.newInstance(positions.map { it.toCell() }.toMutableSet())
 
         positions.map {
             cellManager.changeCell(it.toMineCell())
         }
-        assertThat(cellManager.getSize())
+        assertThat(cellManager.size)
             .isEqualTo(positions.size)
     }
+
+    @DisplayName(value = "전체 open을 해도, 전체 사이즈는 같아야 한다.")
+    @Test
+    fun openAllTestSameSize() {
+        val positions: MutableSet<Position> = mutableSetOf(
+            Position(1, 2),
+            Position(1, 2),
+            Position(1, 3)
+        )
+        val cellManager = CellManager.newInstance(positions.map { it.toCell() }.toMutableSet())
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val positions: Set<Position> = setOf(
            Position(1, 2),
            Position(1, 2),
            Position(1, 3)
        )
        val cellManager = CellManager.newInstance(positions.map { it.toCell() }.toSet())
```